events/handler_test.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package events
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/cadvisor/utils/oomparser"
+)
+
+func createOldTime(t *testing.T) time.Time {
+	const longForm = "Jan 2, 2006 at 3:04pm (MST)"
+	linetime, err := time.Parse(longForm, "Feb 3, 2013 at 7:54pm (PST)")
+	if err != nil {
+		t.Fatalf("could not format time.Time object")
+	} else {
+		return linetime
+	}
+	return time.Now()
+}
+
+func createFakeOom(inTime time.Time) *oomparser.OomInstance {
+	return &oomparser.OomInstance{
+		ContainerName: "/",
+		Pid:           5,
+		ProcessName:   "fakeOomInstance",
+		TimeOfDeath:   inTime,
+	}
+}
+
+// used to convert an OomInstance to an Event object
+func oomInstanceToEvent(currentOomInstance *oomparser.OomInstance) *Event {
```

Comment:
We can simplify this by not using an oomparser.OomInstance. Since events doesn't care what's inside we can take ContainerName and Timestamp as arguments, make the EventType TypeOom and make EventData int(0)